zwave_js_server/model/controller/__init__.py

```diff
@@ -793,7 +794,7 @@ async def async_firmware_update_ota(
             },
             require_schema=24,
```

Comment:
Doesn't this now require schema 29? Otherwise it can't parse the return data (see my comment on server PR too).

Reply:
the require_schema attribute is effectively a no-op now that the minimum schema version matches the maximum schema version, but you are right

Reply:
True, I know, but I just thought that if it exists, then we should use it properly.